code/modules/admin/topic.dm

```diff
@@ -3019,6 +3019,59 @@
 		else
 			error_viewer.showTo(usr, null, href_list["viewruntime_linear"])
 
+	// Library stuff
+	else if(href_list["library_book_id"])
+		var/isbn = href_list["library_book_id"]

```

Comment:
If an admin gets fed a bad topic datum, could they not be fed an arbitrary isbn, and thus possibly an unsanitized arbitrary query?